doc/api/modules.md

```diff
@@ -171,12 +171,17 @@ relative, and based on the real path of the files making the calls to
 ## Loading ECMAScript modules using `require()`
 
 <!-- YAML
+added:
+  - v22.0.0
+  - v20.17.0
 changes:
   - version: REPLACEME
     pr-url: https://github.com/nodejs/node/pull/55085
-    description: require() now supports loading synchronous ES modules by default.
+    description: This feature is no longer behind the `--experimental-require-module` CLI flag.
 -->
 
+> Stability: 1 - Experimental
```

Comment:
```suggestion
> Stability: 1.2 - Release candidate
```

I don't think this is particularly matching

> * 1.1 - Active development. Experimental features at this stage are nearing
>   minimum viability.

And

> * 1.2 - Release candidate. Experimental features at this stage are hopefully
>   ready to become stable. No further breaking changes are anticipated but may
>   still occur in response to user feedback. We encourage user testing and
>   feedback so that we can know that this feature is ready to be marked as
>   stable.

looks close.